diff --git a/widgets/msglist.go b/widgets/msglist.go
index 827282b..8f5a06e 100644
--- a/widgets/msglist.go
+++ b/widgets/msglist.go
@@ -102,11 +102,14 @@ func (ml *MessageList) Draw(ctx *ui.Context) {
 			continue
 		}
 
-		uiConfig := ml.conf.GetUiConfig(map[config.ContextType]string{
+		confParams := map[config.ContextType]string{
 			config.UI_CONTEXT_ACCOUNT: ml.aerc.SelectedAccount().AccountConfig().Name,
 			config.UI_CONTEXT_FOLDER:  ml.aerc.SelectedAccount().Directories().Selected(),
-			config.UI_CONTEXT_SUBJECT: msg.Envelope.Subject,
-		})
+		}
+		if msg.Envelope != nil {
+			confParams[config.UI_CONTEXT_SUBJECT] = msg.Envelope.Subject
+		}
+		uiConfig := ml.conf.GetUiConfig(confParams)
 
 		msg_styles := []config.StyleObject{}
 		// unread message
